### DISCUSSION TITLE
Style external elements with specific class name

### DISCUSSION THREAD

**Discussion by Gytjarek** [chakra-ui/panda#2509] [Q&A] [answered]
I've defined a button recipe that I use throughout my app, and it's all good. However, I want the ability to apply my button styles to an element in the DOM with a specific class name, which I don't have access to, as it's a button created by a third-party library. Is that possible? I know I can define the styles in a global css, but this is not ideal approach

**Comment by anubra266** [comment, marked as answer]
@Gytjarek You might be looking for this
https://panda-css.com/docs/guides/static#generating-recipes

Example
https://play.panda-css.com/kVA5otYUmt
